bindings/csharp/Keystone.Net/Engine.cs

```diff
@@ -3,150 +3,160 @@
 using System.Runtime.InteropServices;
 using System.Threading;
 
-namespace KeystoneNET
+namespace Keystone
```

Comment:
Your renaming (this and the others) makes sense to me but you're broking everything (as you said).
Anyhow, for me it's ok as long it's ok also for @aquynh that maybe has a clearer visibility on how many people are using this.

Reply:
Here is why I made these changes:
- [There shouldn't be uppercase parts in names](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions), and I think `Keystone` looks better than `KeystoneNet`. It conflicts with the [Keystone Framework](https://www.nuget.org/packages/Keystone), but to me it doesn't matter, since their [toplevel namespace doesn't contain many members](http://www.keystoneframework.com/docs/index.html#frlrfKeystone.html) (thus, no conflict in code).
- It's a little redundant to specify `KeystoneEngine` as opposed to `Engine`, for example. Plus `Engine` conveys the purpose better.
- The library has never been published on NuGet, which means that everyone using it has gotten it from GH. As such, major changes such as this one are expected, and should hapen sooner rather than later.
- Some enums and enum members haven't been renamed because they're not used directly in wrapped, managed types. Maybe I should change this?

Reply:
No need to explain, for me it was already ok. Just see what @aquynh think.

Reply:
[He gave his opinion there](https://github.com/keystone-engine/keystone/pull/344#issuecomment-377777761), so the decision is yours to make.